bigquery/indexer.py

```diff
@@ -238,24 +205,25 @@ def index_table(es, bq_client, storage_client, index_name, table,
     # Wait up to 10 minutes for the resulting export files to be created.
     job.result(timeout=600)
     if sample_id_column in [f.name for f in table.schema]:
-        scripts_by_id = _sample_scripts_by_id_from_export(
+        _add_sample_table_to_participant_docs(
```

Comment:
Have _add_sample_table_to_participant_docs() and _add_participant_table_to_participant_docs() return participant_docs, it's a little more clear

```
participant_docs = _add_sample_table_to_participant_docs(...
```